src/keydra/providers/aws_kinesisfirehose.py

```diff
@@ -56,6 +64,14 @@ def _distribute(self, secret, target):
         except Exception as e:
             raise DistributionException(e)
 
+        LOGGER.info(
+            'Successfully distributed {} to Kinesis Firehose'.format(
+                target['key']
```

Comment:
Looks like `target['key']` is the name of the Kinesis stream.. Probably meant `secret['key']`? Or both `secret['key']` and `secret['source']`. Would actually be nice to log this in a single place for all providers, e.g. https://github.com/Athena-Home-Loans/keydra/blob/main/src/keydra/keydra.py#L200 :)

Reply:
Fixed, update inbound.